db/db_options_test.cc

```diff
@@ -289,6 +289,32 @@ TEST_F(DBOptionsTest, SetDelayedWriteRateOption) {
   ASSERT_EQ(20000, dbfull()->TEST_write_controler().max_delayed_write_rate());
 }
 
+TEST_F(DBOptionsTest, MaxTotalWalSizeChange) {
+  Random rnd(1044);
+  const auto key_size = size_t(1024);
+  const auto key = test::RandomKey(&rnd, key_size);
```

Comment:
I pointed you to the wrong function, but this is a value instead of a key. Rename the variable and use test:: RandomString instead?